blueprints/data-solutions/vertex-mlops/main.tf

```diff
@@ -190,19 +190,19 @@ module "cloudnat" {
 
 module "project" {
   source          = "../../../modules/project"
-  name            = var.project_id
-  parent          = try(var.project_create.parent, null)
-  billing_account = try(var.project_create.billing_account_id, null)
-  project_create  = var.project_create != null
+  name            = var.project_config.project_id
+  parent          = try(var.project_config.parent, null)
+  billing_account = try(var.project_config.billing_account_id, null)
```

Comment:
this `try()` is not needed and can hide other issues. Make the variable `nullable = false` and provide sensible defaults